### ISSUE TITLE
Cross platform development

### ISSUE THREAD

**Issue by nholland20** [Bungalow64/DBConfirm#133]
Hi - is this project designed for cross platform development? We are looking for a solution to move our db unit tests to Linux.

**Comment by jamie-burns** [comment]
Hey - thanks for your question. This project is written in .net and that runs on both Windows and Linux. You could set up a C# project that runs the DB tests and deploy that to Linux, if that's what you're meaning?

**Comment by nholland20** [comment]
Yeah, we are looking to move our build and test machines from Windows and Linux.
Another quick question. We are trying to find a way to both build cross-platform and something that works for a database engineers as well. Have you considered adding database set up as direct sql or is that out-of-scope for this project?

**Comment by jamie-burns** [comment]
Do you mean a setup script that runs once before the test run actually starts? If so, then that's probably best handled as a separate step before you kick the tests off. This project is mainly geared at making the individual tests repeatable, so there's nothing in there for the overall test run start or end.

If you look at the CI build for this project (https://github.com/Bungalow64/DBConfirm/blob/main/azure-pipelines.yml) we've got something similar there. The first task is setting up the DBs in Docker, then doing the build and running the tests. You might not be using Docker but the principle should hopefully be the same, whatever you're using to run the tests should hopefully be able to run any setup script.
